heroku/cli
# refactor: migrate orgs:open to oclif/core

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Run `./bin/run orgs:open -t TEAM-NAME`
    - If you are logged in, a browser window should open to the team on Heroku
    - If you are not logged in, a browser window should open to the login page. Once you log in, it will not take you to the team page.
    - You should see a message in the terminal that looks like this:
<img width="699" alt="image" src="https://github.com/heroku/cli/assets/12739849/3c1bd771-f588-47ea-9fe8-01f3fc204b08">
        - This is different from the current implementation. Running `heroku orgs:open -t TEAM-NAME` will not print a message in the terminal.
